File: 02._data_formtas/homework/node/txtReader.js
import * as fs from "fs";

//read file
fs.readFile("../data/car.txt", "utf-8", (err, data) => {
  if (err) {
    throw err;
  } else {
    // converting data to strings
    const stringData = data;

    //assigning data to the variables
    const [brand, price, availableColors] = stringData
      .split("\n")
      .map((line) => line.trim());

    // creating the car object
    const carDetails = {
      brand,
      price,
      availableColors: availableColors.split(","),
    };

    console.log("TEXT FILE");
    console.log(carDetails);
  }
});

File: 02._data_formtas/homework/python/txtReader.py
with open("../data/car.txt") as file:

  data = file.read()

  brand, price, availableColors = data.strip().split("\n")

  carObject = {
    "brand": brand,
    "price": price,
    "availableColors": availableColors.split(',')
  }

  print('TXT FILE')
  print(carObject)


